.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            // Use IntelliSense to find out which attributes exist for C# debugging
+            // Use hover for the description of the existing attributes
+            // For further information visit https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger-launchjson.md
+            "name": ".NET Core Launch (console)",
+            "type": "coreclr",
+            "request": "launch",
+            "preLaunchTask": "build",
+            // If you have changed target frameworks, make sure to update the program path.
+            "program": "${workspaceFolder}/EchoServer/bin/Debug/net6.0/EchoServer.dll",
+            "args": [],
+            "cwd": "${workspaceFolder}/EchoServer",
+            // For more information about the 'console' field, see https://aka.ms/VSCode-CS-LaunchJson-Console
+            "console": "internalConsole",
+            "stopAtEntry": false
+        },
+        {
+            "name": ".NET Core Attach",
+            "type": "coreclr",
+            "request": "attach"
+        }
+    ]
+}
```

Comment:
this is an odd rule, but a rule none the same.  Text files are expected to have an EOF character.  A linter normally helps to keep this straight, but I am not suggesting you set up a linter.  I actually believe working without one helps you learn how to write cleaner formatted code naturally.

I digress, there are several files that have this issue. You may want to do a minor cleanup to resolve.

Reply:
<img width="662" alt="Screen Shot 2022-05-18 at 2 43 24 PM" src="https://user-images.githubusercontent.com/73620216/169143051-f85c3bfa-7e62-4ffb-bda0-7393e837e14e.png">

So when I add the files to push up to GitHub, it gives me this warning - on my local machine, I still have the extra line.  So do we think it's something I need to figure out in my Settings so that it doesn't remove it with this warning?  Or is this a C# thing?

I just tried to adjust it for this file...I don't know that it accepted it?

Reply:
I've seen this before.  There is a UNIX vs DOS thing in how they handle return characters in a text file.  What I was referring to is essentially .. "you need a black line at the end of every text file".

Reply:
Here is documentation that helps to explain what GIT is doing ....

Reply:
Okay I think I fixed it in my Rider settings.